neural_transport/models/flowmatching.py

```diff
@@ -66,18 +66,21 @@ def __init__(
         for var in reversed(required_input_vars):
             if var not in input_vars:
                 input_vars.insert(0, var)
-        model_kwargs["input_vars"] = input_vars
-        model_kwargs["model_kwargs"]["in_chans"] += 1 + model_kwargs["nlev"] * 1 # + 1 for flow_time, + nlev for co2massmix
+        
+        if "model_kwargs" in model_kwargs:
+            sub_kwargs = model_kwargs["model_kwargs"]
+        else:
+            sub_kwargs = model_kwargs
+
+        sub_kwargs["input_vars"] = input_vars
+        sub_kwargs["in_chans"] += 1 + sub_kwargs["nlev"] * 1 # + 1 for flow_time, + nlev for co2massmix
```

Comment:
Ok i see - to be more generic it should not be `* 1` but `* len(target_vars)` or not?
I usually prefer to keep these types of calculations as a task for the user, because it forces them to at least think a bit about what they are running - but I also see the appeal of automatically altering things like this here